server/core/kv_test.go

```diff
@@ -157,6 +158,22 @@ func (s *testKVSuite) TestLoadRegionsExceedRangeLimit(c *C) {
 	}
 }
 
+func (s *testKVSuite) TestLoadGCSafePoint(c *C) {
+	kv := NewKV(NewMemoryKV())
+	testData := []uint64{0, 1, 2, 233, 2333, 23333333333, math.MaxUint64}
+
+	r, e := kv.LoadGCSafePoint()
+	c.Assert(r, Equals, uint64(0))
+	c.Assert(e, IsNil)
+	for _, safePoint := range testData {
+		err := kv.SaveGCSafePoint(safePoint)
+		c.Assert(err, IsNil)
+		safePoint1, err := kv.LoadGCSafePoint()
+		c.Assert(err, IsNil)
+		c.Assert(safePoint, Equals, safePoint1)
+	}
```

Comment:
Any need to construct a test case that safePoint is less than previous safePoint?

Reply:
oh, I found it in client test.